clang/test/dpct/asm/prefetch_default.cu

```diff
@@ -0,0 +1,70 @@
+// UNSUPPORTED: cuda-8.0
+// UNSUPPORTED: v8.0
+// RUN: dpct --format-range=none -out-root %T/prefetch_default %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/prefetch_default/prefetch_default.dp.cpp
+// RUN: %if BUILD_LIT %{icpx -c -DBUILD_TEST -fsycl %T/prefetch_default/prefetch_default.dp.cpp -o %T/prefetch_default/prefetch_default.dp.o %}
+
+// clang-format off
+#include <cuda_runtime.h>
+
+__global__ void prefetch(int *arr) {
+#ifndef BUILD_TEST
+  /* prefetch of no address space */
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.L1 [%0];" : : "l"(arr));
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.L2 [%0];" : : "l"(arr));
+
+  /* prefetch of global address space */
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.global.L1 [%0];" : : "l"(arr));
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.global.L2 [%0];" : : "l"(arr));
+
+  /* using Register-Immediate (Displacement) address mode */
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile("prefetch.global.L1 [%0 + 2];" :: "l"(arr));
+
+  /* prefetch of global address space with eviction priority */
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.global.L2::evict_last [%0];" : : "l"(arr));
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1053:{{.*}} Migration of device assembly code is not supported.
+  // CHECK-NEXT: */
+  asm volatile ("prefetch.global.L2::evict_normal [%0];" : : "l"(arr));
```

Comment:
Could be supported,  [prefetch_default.cu](https://github.com/oneapi-src/SYCLomatic/pull/2579/files#diff-edb9f319de5415c9a48776eedc82d861424ad2e81c86e04cec92e6fca350e6f4) should keep the same contents as [prefetch.cu](https://github.com/oneapi-src/SYCLomatic/pull/2579/files#diff-167760ebe25316fac283a9e0448236fe7718ffdc83cd0bc8feedf479b04072ee)

Reply:
Done